Combination of Fedora's zoom-0.4.1-authors.patch and
zoom-0.4.1-cert-chain-must-not-be-nil.patch.

--- origsrc/zoom-0.4.1/zoom.gemspec	2012-08-22 21:27:26.651309400 -0500
+++ src/zoom-0.4.1/zoom.gemspec	2012-08-22 21:27:47.099479000 -0500
@@ -6,6 +6,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = nil if s.respond_to? :required_rubygems_version=
   s.require_paths = ["lib"]
   s.autorequire = "zoom"
+  s.authors = ["Laurent Sansonetti"]
   s.date = "2007-11-19"
   s.extensions = ["src/extconf.rb"]
   s.files = ["src/rbzoomquery.c", "src/rbzoomresultset.c", "src/rbzoomoptions.c", "src/rbzoompackage.c", "src/rbzoomrecord.c", "src/rbzoom.c", "src/rbzoomconnection.c", "src/rbzoom.h", "src/extconf.rb", "test/zebra", "test/record.dat", "test/search_test.rb", "test/search_batch_test.rb", "test/thread_test.rb", "test/package_live.rb", "test/package_test.rb", "test/record.txt", "test/record-update.xml", "test/record.xml", "test/zebra/records", "test/zebra/register", "test/zebra/shadow", "test/zebra/tab", "test/zebra/key", "test/zebra/lock", "test/zebra/zebra.cfg", "test/zebra/records/programming_ruby.xml", "test/zebra/records/programming_ruby_update.xml", "test/zebra/register/empty_file", "test/zebra/shadow/empty_file", "test/zebra/tab/string.chr", "test/zebra/tab/record.abs", "test/zebra/tab/bib1.att", "test/zebra/tab/default.idx", "test/zebra/tab/usmarc.mar", "test/zebra/tab/numeric.chr", "test/zebra/key/empty_file", "test/zebra/lock/empty_file", "sample/hello.rb", "sample/needle.rb", "README", "ChangeLog", "Rakefile"]
